### PR TITLE
Fixed display flex issue

### PR DESCRIPTION
**Pull request by noahc3**

### CODE REVIEW

**Review by travisfriesen — APPROVED** [review]
lgtm!

**Review by pieberrykinnie — COMMENTED** [review]
bad change inline css >:(

File: src/pages/Resources/Resources.tsx [hunk 1]
```diff
@@ -124,7 +124,7 @@ export const Resources = () => {
         return(
             <Box fontSize={'sm'}>
                 <li>
-                    <HStack>
+                    <HStack style={{'display': 'inline-flex'}}>
```
Comment (noahc3) [inline src/pages/Resources/Resources.tsx:127]:
```suggestion
                    <HStack style={'display': 'inline-flex'}>
```

fix blahb lah

**Review by pieberrykinnie — APPROVED** [review]

**Comment by DucharmeEthan** [comment]
LGTM

**Pull request merged**